File: examples/python/src/generate_custom_metric.py
import time
from platform_os import is_windows

if is_windows():
    from request import post
else:
    from sock import post

metrics = [
    {
        "key": "test-key" + str(x),
        "value": 10.52 + x,
        "occurred_at": int(time.time() * 1000 + x),
    }
    for x in range(10)
]


json_response = post("/custom-metrics", metrics)

print("The response is as follows.\n")
print(json_response)

File: examples/nodejs/src/generateCustomMetric.js
import { post } from "./lib/sock.js";

(async () => {
  const payload = {
    key: "test-key",
    value: 10.52,
    occurred_at: Date.now(),
  };
  const response = await post("/custom-metrics", payload);

  console.log("The response is as follows.\n");
  console.log(response);
})();
